main.go

```diff
@@ -37,27 +37,28 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-// After running the executable, you may connect to it using the following:
-//
-// > mysql --host=localhost --port=3306 --user=root
-//
-// The included MySQL client is used in this example, however any MySQL-compatible client will work.
-
 var (
 	initMode = false
 
 	address       = "0.0.0.0"
 	port          = 3306
 	socket        string
+	defaultDb     = "mysql"
```

Comment:
Great! Does this mean we can switch to a different default database name by simply changing this variable? If so, maybe we can switch to `myduck` in this PR.

Reply:
No, we don't support it now. This feature is not currently supported. The changes in this PR lay the groundwork for its future implementation.

Reply:
Get it.